.github/workflows/e2e-pull-request.yml

```
@@ -30,11 +30,13 @@ jobs:
     strategy:
       fail-fast:     false
       matrix:
-        test_groups:   [ 'wcpay', 'subscriptions', 'upe' ] # [TODO] Unskip blocks tests after investigating constant failures.
+        test_groups:   [ 'wcpay', 'subscriptions', 'upe', 'upeSplit' ] # [TODO] Unskip blocks tests after investigating constant failures.
         test_branches: [ 'merchant', 'shopper' ]
         exclude:
           - test_groups: 'upe'
-            test_branches: 'merchant' 
+            test_branches: 'merchant'
+          - test_ groups: 'upeSplit'
```

This typo will make the e2e test suite

```suggestion
          - test_groups: 'upeSplit'
```